cms/categories/models.py

```diff
@@ -26,7 +26,35 @@ class CategoryPageCategoryRelationship(models.Model):
 class CategoryPage(Page):
     """CategoryPages are pages which have categories, like Blogs and Publications."""
 
-    pass
+    def blurb(self, length=110):  # Length empirically chosen, feel free to change.
+        """
+        Create a short extract for displaying on the search page.
+        We would call it an excerpt or a snippet, but both those names are taken
+        (by a page field and a wagtail concept of smaller-than-page fragments)
+        """
+
+        body = self.specific.body
+        # Later we'll probably want to handle stream fields but this'll do for now.
+        assert (
```

Comment:
This feels very much like something which should throw its own exception type, eg:

``` python
class CannotBlurbNonString(Exception):
    pass

if type(body) != str
    raise CannotBlurbNonString("The body of this page is not a plain string")
```

I'd also tweak the error message so it describes the logic better; the problem isn't that the body is richtext, the problem is that it isn't a string.